src/features/network-modifications/twoWindingsTransformer/tapChanger/phaseTapChanger/PhaseTapChangerPaneSteps.tsx

```
@@ -0,0 +1,180 @@
+            [FieldConstants.STEPS_SUSCEPTANCE]: parseIntData(
+                val[
+                    intl.formatMessage({
+                        id: 'ImportFileSusceptance',
+                    })
+                ],
+                0
+            ),
+            [FieldConstants.STEPS_RATIO]: isNaN(parseFloat(val[intl.formatMessage({ id: 'Ratio' })]))
+                ? 1
+                : parseFloat(val[intl.formatMessage({ id: 'Ratio' })]),
+            [FieldConstants.STEPS_ALPHA]: isNaN(
```

ESLint

[error] Unexpected use of 'isNaN'. Use Number.isNaN instead https://github.com/airbnb/javascript#standard-library--isnan

(no-restricted-globals)